catalog/v4.12/openshift-builds-operator/catalog.yaml

```
@@ -4,21 +4,54 @@ name: openshift-builds-operator
 schema: olm.package
 ---
 entries:
-- name: openshift-builds-operator.v1.0.0
-- name: openshift-builds-operator.v1.0.1
-- name: openshift-builds-operator.v1.0.2
-  skips:
-  - openshift-builds-operator.v1.0.0
-  - openshift-builds-operator.v1.0.1
-- name: openshift-builds-operator.v1.1.0
-  replaces: openshift-builds-operator.v1.0.2
-  skips:
-  - openshift-builds-operator.v1.0.0
-  - openshift-builds-operator.v1.0.1
+  - name: openshift-builds-operator.v1.0.0
+  - name: openshift-builds-operator.v1.0.1
+  - name: openshift-builds-operator.v1.0.2
+    skips:
+    - openshift-builds-operator.v1.0.0
+    - openshift-builds-operator.v1.0.1
+  - name: openshift-builds-operator.v1.1.0
+    replaces: openshift-builds-operator.v1.0.2
+    skips:
+      - openshift-builds-operator.v1.0.0
+      - openshift-builds-operator.v1.0.1
 name: latest
 package: openshift-builds-operator
 schema: olm.channel
 ---
+entries:
+  - name: openshift-builds-operator.v1.0.0
+  - name: openshift-builds-operator.v1.0.1
+  - name: openshift-builds-operator.v1.0.2
+    skips:
+      - openshift-builds-operator.v1.0.0
+      - openshift-builds-operator.v1.0.1
+  - name: openshift-builds-operator.v1.1.0
+    replaces: openshift-builds-operator.v1.0.2
+    skips:
+      - openshift-builds-operator.v1.0.0
+      - openshift-builds-operator.v1.0.1
+name: stable-v1
+package: openshift-builds-operator
+schema: olm.channel
+---
+entries:
+  - name: openshift-builds-operator.v1.1.0
+name: openshift-builds-1.1
```

Per discussion in Slack, rename this to `builds-1.1`

```suggestion
name: builds-1.1
```

@sayan-biswas We need to add a release note that we are changing the channel naming convention to "builds-1.N" for z-stream only updates.

---

updated!